content/docs/reference-javascript-environment-requirements.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: javascript-environment-requirements
-title: JavaScript Environment Requirements
+title: Kebutuhan Lingkungan JavaScript
 layout: docs
 category: Reference
 permalink: docs/javascript-environment-requirements.html
 ---
 
-React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). If you support older browsers and devices which may not yet provide these natively (e.g. IE < 11) or which have non-compliant implementations (e.g. IE 11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
+React 16 membutuhkan tipe data *collection* seperti [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) dan [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). Jika kamu ingin mendukung browser dan device lama yang mungkin belum menyediakannya secara langsung (contoh: IE < 11) atau browser yang mengimplementasikannya dengan tidak sesuai (contoh: IE 11), mohon menambahkan *polyfill global* di bundel aplikasi, seperti [core-js](https://github.com/zloirock/core-js) atau [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```

Comment:
device -> perangkat
browser -> *browser*

I'd add it to the glossary 😄

Reply:
noted 👍

Reply:
done